internal/parse/parse.go

```diff
@@ -101,10 +102,14 @@ func GetBindMount(ctx *types.SystemContext, args []string, contextDir string, st
 			if len(kv) == 1 {
 				return newMount, "", fmt.Errorf("%v: %w", kv[0], errBadOptionArg)
 			}
-			if err := parse.ValidateVolumeCtrDir(kv[1]); err != nil {
+			targetPath := kv[1]
+			if !path.IsAbs(targetPath) {
+				targetPath = filepath.Join(workDir, targetPath)
```

Comment:
Should this path be resolved?

filepath.EvalSymlinks
Or at least
filepath.Clean?

Reply:
Yes I think target path must be cleaned before join, since things is executed on host side symlink should still be allowed I think.

Reply:
I don't think we want to allow symlinks, as noted elsewhere, they're evil....

Reply:
This part is only executed on host side so there is no room for remote execution of symlink hence they are allowed just like we allow it for all the other mounts already.